export.go

```diff
@@ -8,7 +8,7 @@ import (
 // exportBufferSize is the number of nodes to buffer in the exporter. It improves throughput by
 // processing multiple nodes per context switch, but take care to avoid excessive memory usage,
 // especially since callers may export several IAVL stores in parallel (e.g. the Cosmos SDK).
-const exportBufferSize = 32
+const exportBufferSize = 64
```

Comment:
could you provide some benachmarks to compare the difference please

Reply:
chasing another bug.

Reply:
gonna throw into draft until we can get a benchmark showing the difference